File: extensions/CData ADO.NET Provider for MongoDB 2015/demos - winform/querybrowser/cs/Program.cs
using System;
using System.Collections.Generic;
using System.Globalization;
using System.Threading;
using System.Windows.Forms;
using System.IO;

namespace CData.MongoDB.Demo {
  static class Program {
    /// <summary>
    /// The main entry point for the application.
    /// </summary>
    [STAThread]
    static void Main() {
      Application.EnableVisualStyles();
      Application.SetCompatibleTextRenderingDefault(false);
      Application.Run(new GenerationForm());
    }
  }
}

File: extensions/CData ADO.NET Provider for MongoDB 2015/demos - winform/querybrowser/vb/Program.vb
Imports System.Collections.Generic
Imports System.Windows.Forms
Imports System.IO

Namespace CData.MongoDB.Demo
  NotInheritable Class Program
    Private Sub New()
    End Sub
    ''' <summary>
    ''' The main entry point for the application.
    ''' </summary>
    <STAThread> _
    Private Shared Sub Main()
      Application.EnableVisualStyles()
      Application.SetCompatibleTextRenderingDefault(False)
      Application.Run(New GenerationForm())
    End Sub
  End Class
End Namespace